products/devstudio/_common/product.yml

```diff
@@ -1,6 +1,6 @@
 name: Red Hat JBoss Developer Studio
 abbreviated_name: JBoss Developer Studio
-current_version: 8.0.0.GA
+current_version: 8.1.0.Beta1
```

Comment:
This is the current GA release. If you change it for pre-GA releases, it causes two problems:

1. The big green download button on the right points to the Beta release
2. You delegate the GA release to the "more downloads" area on the downloads page

See: http://www-stg.jboss.org/pr/817/build/1781/products/devstudio/download/

Reverting this line change will fix the issue. The site build will automatically detect a pre-GA release that is newer than the latest GA release and show it on http://www-stg.jboss.org/pr/817/build/1781/products/devstudio/download/.